packages/not-next/src/hooks/useShopId.ts

```diff
@@ -1,9 +1,9 @@
 import { useEffect, useState } from "react";
 import { useSearch } from "@tanstack/react-router";
-// import { ShopId } from "@/types";
+import { ShopId } from "../types.ts";
 
-export default function useShopId() {
-  const [shopId, setShopId] = useState<bigint | null>(null);
+export function useShopId() {
+  const [shopId, setShopId] = useState<ShopId | null>(null);
```

Comment:
at first I didn't really like using `ShopId` instead of `bigint`. But now I think I see the value in the extra level of indirection as it makes changing underlining type easier if needed